cmd/cli/cmd/migrate.go

```diff
@@ -96,28 +116,47 @@ func NewMigrate() *cobra.Command {
 			okCheck := color.New(color.FgGreen).FprintlnFunc()
 			okCheck(cmd.OutOrStdout(), "\nMigration Succeeded 🎉")
 
-			if opts.SkipConnect {
+			instanceURL := fmt.Sprintf("%s/instances/%s", opts.CloudDashboardURL, instanceID)
+
+			if opts.SkipOpenBrowser {
+				fmt.Println(heredoc.Docf(`
+				 Visit the URL to see your instance details:
+				 %s
+		`, instanceURL))
 				return nil
 			}
-			return browser.OpenURL(fmt.Sprintf("%s/instances/%s", opts.CloudDashboardURL, instanceID))
+
+			fmt.Println(heredoc.Docf(`
+			If your browser didn't open automatically, visit the URL to see your instance details:
+				 %s
+		`, instanceURL))
+			return browser.OpenURL(instanceURL)
 		},
 	}
 
 	flags := login.Flags()
 	flags.StringVar(&opts.Token, "token", "", "Botkube Cloud authentication token")
 	flags.StringVar(&opts.InstanceName, "instance-name", "", "Botkube Cloud Instance name that will be created")
-	flags.StringVar(&opts.CloudAPIURL, "cloud-api-url", "https://api.botkube.io", "Botkube Cloud API URL")
+	flags.StringVar(&opts.CloudAPIURL, "cloud-api-url", "https://api.botkube.io/graphql", "Botkube Cloud API URL")
 	flags.StringVar(&opts.CloudDashboardURL, "cloud-dashboard-url", "https://app.botkube.io", "Botkube Cloud URL")
 	flags.StringVarP(&opts.Label, "label", "l", "app=botkube", "Label of Botkube pod")
 	flags.StringVarP(&opts.Namespace, "namespace", "n", "botkube", "Namespace of Botkube pod")
 	flags.BoolVarP(&opts.SkipConnect, "skip-connect", "q", false, "Skips connecting to Botkube Cloud after migration")
+	flags.BoolVar(&opts.SkipOpenBrowser, "skip-open-browser", false, "Skips opening web browser after migration")
+	flags.BoolVar(&opts.AutoUpgrade, "auto-upgrade", false, "Automatically upgrades Botkube instance without additional prompt")
```

Comment:
**suggestion**

WDYT about having sth more generic like `--no-input`/`--no-interactivty`/`--no-confirm`?
So basically a user will have one flag to use when they want to disable all prompts, e.g. on CI.

https://clig.dev/#interactivity